### DISCUSSION TITLE
Pulsar Doesn't Fully Close on Quit - Must Force Quit

### DISCUSSION THREAD

**Discussion by jvanbelzen** [pulsar-edit/.github#173] [Pulsar Support]
Pulsar doesn't fully close after CMD+Q or Pulsar>Quit Pulsar.  Active windows are closed, but the program remains open in the dock.  The only way I've been able to fully close Pulsar is by Force Quitting.

This was also a [reported problem](https://github.com/atom/atom/issues/17672) with Atom, so I'm guessing it was inherited. 

Pulsar 1.104.0
Mac M2
MacOS Ventura 13.3.1

**Comment by Daeraxa** [comment]
@DeeDeeG might be interested in this as the person that made the PR to close that linked issue. It sounds like maybe this should be opened as a bug in the Pulsar repo.

**Reply by confused-Techie** [reply]
Yeah @jvanbelzen I'd really recommend opening this as a bug in the Pulsar repo. So that way it gets better visibility, and can be properly tracked.

Discussions in Pulsar support are more intended for "How do I do X, Y and Z" rather than an actual bug report. But thanks a ton for bringing this to our attention!
